components/brave_rewards/browser/rewards_p3a.h

```diff
@@ -73,6 +73,19 @@ void RecordNoWalletCreatedForAllMetrics();
 
 void RecordRewardsDisabledForSomeMetrics();
 
+enum class RewardsEnabledDuration {
+  kNever,
+  kStillEnabled,
+  kHours,
+  kDays,
+  kWeeks,
+  kMonths,
+  kLonger,
```

Comment:
longer than what? mb there could be a better name

Reply:
`kMoreThanMonths`? `kSeasons`?

Reply:
`kQuarters`?

Reply:
Appropriately corporate for "analytics" :)